gcloud/_testing.py

```diff
@@ -35,11 +35,6 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             setattr(self.module, key, value)
 
 
-def _monkey_defaults(*args, **kwargs):
-    mock_defaults = _DefaultsContainer(*args, **kwargs)
-    return _Monkey(_helpers, _DEFAULTS=mock_defaults)
-
-
 def _setup_defaults(test_case, *args, **kwargs):
     test_case._replaced_defaults = _helpers._DEFAULTS
     _helpers._DEFAULTS = _DefaultsContainer(*args, **kwargs)
```

Comment:
OIC.  I misfired on the `_setup_defaults` beneath it, which seems like it should go away.